src/core_modules/capture-core/metaDataStoreLoaders/index.js

```diff
@@ -1,3 +1,3 @@
 // @flow
-export { default as loadMetaData } from './baseLoader/metaDataLoader';
+export { loadWithContext as loadMetaData } from './baseLoader';
```

Comment:
Thumbs up, absolutely agree. Renaming.

Reply:
On second thought, I will rename the file loadWithContext to loadMetaData and loadMetaData to loadMetaDataInternal and expose the loadMetaData function from loadMetaData.

Temporary will use filename loadMetaDataMain for loadMetaData to avoid some merge trouble and fix it in the last PR.

Reply:
This will be followed up in the last PR do understand you well?

Reply:
yes